components/script/dom/htmlmediaelement.rs

```diff
@@ -835,22 +856,22 @@ impl FetchResponseListener for HTMLMediaElementContext {
         }
         // => "Once the entire media resource has been fetched..."
         else if status.is_ok() {
-            elem.change_ready_state(HAVE_ENOUGH_DATA);
+            elem.change_ready_state(ReadyState::HaveEnoughData);
 
             elem.upcast::<EventTarget>().fire_event(atom!("progress"));
 
-            elem.network_state.set(NETWORK_IDLE);
+            elem.network_state.set(NetworkState::Idle);
 
             elem.upcast::<EventTarget>().fire_event(atom!("suspend"));
         }
         // => "If the connection is interrupted after some media data has been received..."
-        else if elem.ready_state.get() != HAVE_NOTHING {
+        else if elem.ready_state.get() != ReadyState::HaveEnoughData {
```

Comment:
This should be `HaveNothing`, right? Or is this change intentional?

Reply:
REPLACING IS HARD